.stylelintrc

```diff
@@ -0,0 +1,8 @@
+{
+  "extends": "stylelint-config-standard",
+  "ignoreFiles": "extension/css/vendor/*.css",
+  "rules": {
+    "font-family-no-missing-generic-family-keyword": null,
```

Comment:
This "fixed" the following error:

```
extension/popup/popup.css
 15:16  ✖  Unexpected missing generic font family   font-family-no-missing-generic-family-keyword
```

This seems to be coming from the following rule, which I'm guessing doesn't like that there isn't a fallback font:

```css
body,
.cls-1 {
  font-family: "Inter" !important;
}
```

🤷‍♂